frontend/src/app/models/auth/auth-user-dto.ts

```diff
@@ -0,0 +1,7 @@
+import { TokenDto } from '../token/token-dto';
+import { UserDto } from '../user/user-dto';
+
+export interface AuthUserDto{
```

Comment:
Do `ng lint` before committing to be able you do not have any warning
```suggestion
export interface AuthUserDto {
```